install/gcp/installer/forseti_server_installer.py

```diff
@@ -217,8 +218,14 @@ def get_rule_default_values(self):
         Returns:
             dict: A dictionary of default values.
         """
-        organization_id = self.resource_root_id.split('/')[-1]
-        domain = gcloud.get_domain_from_organization_id(organization_id)
+        if self.target_project_id:
+            organization_id = gcloud.lookup_organization(
+                self.target_project_id)
+            domain = gcloud.get_domain_from_organization_id(organization_id)
```

Comment:
the domain var seems to be set the same across both if/else.. should it be moved out?

Reply:
Agreed. Sorry I missed this.